libevmasm/SemanticInformation.cpp

```diff
@@ -132,6 +132,22 @@ bool SemanticInformation::altersControlFlow(AssemblyItem const& _item)
 	}
 }
 
+bool SemanticInformation::terminatesControlFlow(AssemblyItem const& _item)
+{
+	if (_item.type() != Operation)
```

Comment:
Perhaps we should rather assert in this case.

Reply:
Every other function in this class also starts with this. Why an assert here?